app/src/main/java/com/example/bigowlapp/activity/LoginPageActivity.java

```diff
@@ -25,7 +25,7 @@
     TextView tvSignUp;
     private FirebaseAuth.AuthStateListener mAuthStateListener;
     private LogInViewModel logInViewModel;
-    private AuthFailureNotificationListener authListener;
+    private NotificationListenerManager authListener;
```

Comment:
should also change the variable name itself, still called `authListener`

Reply:
done.